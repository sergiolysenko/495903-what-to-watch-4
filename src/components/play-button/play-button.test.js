import React from "react";
import renderer from "react-test-renderer";
import PlayButton from "./play-button.jsx";
import {Router} from "react-router-dom";
import history from "./../../history.js";

it(`Render PlayButton`, () => {
  const tree = renderer.create(
      <Router history={history}>
        <PlayButton
          id={1}
        />
      </Router>).toJSON();
  expect(tree).toMatchSnapshot();
});
