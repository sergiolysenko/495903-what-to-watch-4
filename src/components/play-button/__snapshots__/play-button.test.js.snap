// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PlayButton 1`] = `
<a
  className="btn btn--play movie-card__button"
  href="/films/player/1"
  onClick={[Function]}
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</a>
`;
